http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeV2ServiceInterp.scala

```diff
@@ -347,11 +347,9 @@ class RuntimeV2ServiceInterp[F[_]: Parallel](
                     DiskCannotBeDeletedWsmException(disk.id, wsmState, disk.cloudContext, ctx.traceId)
                   )
                 _ <- persistentDiskQuery.markPendingDeletion(diskId, ctx.now).transaction
-              } yield if (wsmState.isDeleted) F.pure(none[DiskId]) else Some(diskId)
-            // if disk hasn't been created in WSM, don't pass id to back leo
-            case None => F.pure(none[DiskId])
-          }
-        } yield Some(diskId)
+              } yield if (wsmState.isDeleted) None else Some(diskId)
```

Comment:
just make sure the if statement's return here isn't `any`, seems to have shifted from `if (wsmState.isDeleted) F.pure(none[DiskId]) else Some(diskId)` to `if (wsmState.isDeleted) None else Some(diskId)`

Reply:
yeah before the yield was returning any, but we didn't use the value so it didn't matter, but I fixed it so I could use it (ie its not `any` anymore)